learning-bitmark/quick-start/simple-solution-for-node-setup.md

```diff
@@ -1 +1,67 @@
 #  [Learning Bitmark / Quick start] - Simple solution for node setup
+
+### Start your bitmark-node
+This section will quide you to start a bitmark docker node. The **docker** node, also called **bitmark-node**, is a bitmark full-node. A full-node can generate blocks, mine the blocks and do verification of payments. 
```

Comment:
Does `bitmark-node` only refer to the node running inside a docker container?

Reply:
Do you think we should introduce a bit about the bitmark node in general first, then back to the content of the section? 
Just for reference, here is the text from the technical writer for the section `Run a node` which will be removed from the website:
```Unlike conventional property systems that rely on centralized gatekeepers, the Bitmark blockchain is an open and transparent property system that is strengthened through the active participation of anyone on the Internet. You can support the integrity of Bitmark’s open-source blockchain by running the Bitmark node software and becoming a member of its peer-to-peer network. You will earn money and property rewards by doing so.
```

After that introduction, we could mention this section, such as:
```There are different ways to run a Bitmark node, in this section, we introduce a simple way to do so with running a bitmark docker node. ...```

Reply:
Is this ok , anything to add !? 
Start your bitmark node
A user can run a bitmark node in several ways. This section will guide you to start a bitmark node using docker container. Inside a bitmark node docker container, there are bitmarkd, recorderd, payment system and a user interface to control and monitor services.